### PR TITLE
feat: We shouldn't use the latest version tag

### PR DESCRIPTION
**Pull request by gomex**
### Description and Example

We should specify the version number instead latest

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

**Review by igorgiumellizup — APPROVED** [review]

**Review by luisoliveirazup — APPROVED** [review]

**Comment by matheusribeirozup** [comment]
Hi @gomex, why is this necessary? What are the problems with maintaining the most up-to-date OS versions of the machines?

**Comment by gomex** [comment]
@matheusribeirozup the problem is that ci/cd should be specific. We need to assure that one execution is unique and idempotent. If you execute this twice and if before the second attempt the github changes the version it will fail without any change of code.

If you need to rollback, you can't execute the old pipeline, because you don't know which version os will be used to build this.

**Comment by matheusribeirozup** [comment]
Hi @gomex now I understand your point. We imagined that it would be the reason for your suggestion and we even discussed it at the last meeting, and we concluded that we'll keep the latest OS version in our CI anyway. Your point is really valid, especially when we talk about dependencies in a project, but we raised some stronger ones, see if you agree:
- Keeping the OS version always up to date reduces the risk of security breaches.
- When a new OS version brings a problem to the project, we must be aware of it as soon as possible, and we must also prioritize its solution. Both requirements are satisfied if our workflows use the latest OS versions.

In short, when we use the latest OS version in our CI, we guarantee that the problems that could happen to the user side, happen on our side, giving us the opportunity to correct them preventively. If you don't agree with any point or want to raise another one, feel free to reopen this issue. We are very grateful for your contribution :).

**Comment by gomex** [comment]
"Keeping the OS version always up to date reduces the risk of security breaches."

Avoid security breach is a good point, but you can have a specific CI to check it, you can use multiple OS to validate it. I can update my PR if you agree.

**Comment by matheusribeirozup** [comment]
Hi @gomex, our CI is already a [little complex](https://github.com/ZupIT/beagle/wiki/Continuous-Integration), adding these validations would be nice but it would increase even more the complexity and difficulty in maintaining it. And considering that we want these possible OS problems to happen in our workflows, we decided to keep it that way. Thanks again for the contribution.
